src/redux/redux-reducers/userstore.jsx

```diff
@@ -0,0 +1,10 @@
+import {createSlice} from '@reduxjs/toolkit';
+//reducer for the user
+const userStore = createSlice({
+  name: 'userStore',
+  initialState: {email:''},
+  reducers: {
+  },
+});
+export const {addMovies} = movies.actions;
```

Comment:
Why do we have the  movies slice under the user?

Reply:
sry for this mistakes, please check now if i fixed them